Dockerfile

```
@@ -6,15 +6,17 @@ FROM node:22-alpine AS base
 WORKDIR /usr/src/app
 
-ENV YARN_DISABLE_GIT_HOOKS=1
 ENV CI=true
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
 
-RUN apk add --no-cache dumb-init g++ make python3
+RUN apk add --no-cache dumb-init=1.2.5-r3 g++=13.2.1_git20231014-r0 make=4.4.1-r2 python3=3.12.8-r1
 
-COPY --chown=node:node yarn.lock .
+RUN corepack enable
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)